server/main.lua

```diff
@@ -1338,7 +1329,7 @@ end)
 RegisterNetEvent('mdt:server:getCallResponses', function(callid)
 	local src = source
 	local Player = QBCore.Functions.GetPlayer(src)
-	if IsPolice(Player.PlayerData.job.name) then
+	if Config.PoliceJobs or Config.AmbulanceJobs(Player.PlayerData.job.name) then

```

Comment:
This code will not work the way you are intending.

This condition is meant to check if the current player's job is police. You are trying to add an "or if player job is ambulance"

The code change you have made reads: "if i have config.policejobs or i have config.ambulancejobs then"
So this code is no longer doing any check for your job, it either will work for everyone or will error out.

Also Config.AmbulanceJobs is a list, not a function. `Config.AmbulanceJobs(Player.PlayerData.job.name)` is trying to call AmbulanceJobs as a function.

A proper fix to this would be to modify the `IsPolice` function:
```
local function IsPoliceOrEms(job)
	for k, v in pairs(Config.PoliceJobs) do
           if job == k then
              return true
            end
         end
         
         for k, v in pairs(Config.AmbulanceJobs) do
           if job == k then
              return true
            end
         end
    return false
end
```